libs/execution_context/ebpf_native.c

```diff
@@ -612,6 +612,11 @@ _ebpf_native_validate_map(_In_ const ebpf_native_map_t* map, ebpf_handle_t origi
     if (_ebpf_native_is_map_in_map(map)) {
         ebpf_native_map_t* inner_map = map->inner_map;
         ebpf_assert(inner_map != NULL);
+        // Code analysis doesn't understand that inner_map is not NULL if _ebpf_native_is_map_in_map() returns true.
+        if (inner_map == NULL) {
```

Comment:
This is covered in PR #1571, but just mentioning -- we should change if check to `__analysis_assume`